moto/ecs/responses.py

```diff
@@ -87,7 +87,10 @@ def list_task_definitions(self):
     def describe_task_definition(self):
         task_definition_str = self._get_param("taskDefinition")
         data = self.ecs_backend.describe_task_definition(task_definition_str)
-        return json.dumps({"taskDefinition": data.response_object, "failures": []})
+        resp = {"taskDefinition": data.response_object, "failures": []}
+        if "TAGS" in self._get_param("include", []):
+            resp["tags"] = self.ecs_backend.task_definitions[data.family][data.revision].tags
```

Comment:
Would be good to have a utility method for this, i.e. `self.ecs_backend.get_tags_for_task(..)`

Reply:
turns out there already is! I found `list_tags_for_resource`